timeline/account.go

```diff
@@ -191,11 +191,11 @@ func (tl *Timeline) LoadAccounts(ids []int64, dataSourceIDs []string) ([]Account
 	WHERE data_sources.id = accounts.data_source_id`
 	args := make([]any, 0, len(ids)+len(dataSourceIDs))
 	if len(ids) > 0 || len(dataSourceIDs) > 0 {
-		q += " AND (" //nolint:goconst
+		q += " AND (" //nolint:goconst,nolintlint
 	}
 	for i, id := range ids {
 		if i > 0 {
-			q += " OR " //nolint:goconst
+			q += " OR " //nolint:goconst,nolintlint
```

Comment:
Why are the `nolintlint` exceptions needed here now but not before? Is it just lack of explanation?

Reply:
It _looks_ as though the linter parses the files in a non deterministic order; and if this file is processed first it needs the `goconst` declaration to prevent `" OR "` being called out as a common static string, but if it's processed second (of the files with these static strings), then it needs the `nolintlint` to prevent it from being called out as an _unnecessary_ `goconst` declaration 🤦‍♂️ 

I feel you when you say "linters are frustrating" 😅 Do you have a preference as to how I handle this? Leave it as it stands; remove the `goconst` linter entirely?

Reply:
Working through these changes and I get this error:

```
timeline/search.go:493:9: string ` OR ` has 12 occurrences, make it a constant (goconst)
                        q += " OR "
                             ^
timeline/account.go:198:16: directive `//nolint:goconst` is unused for linter "goconst" (nolintlint)
                        q += " OR " //nolint:goconst
```

Making these swaps (as I have done in the latest comments) moves the linter to "no issues", but I'm concerned they'll switch back at some random point in the future.